odl/test/trafos/fourier_test.py

```diff
@@ -631,6 +631,53 @@ def char_rect(x):
             assert all_almost_equal(ft.adjoint(ft(char_rect)), discr_rect)
 
 
+def _test_adjoint(operator):
+    """Test adjoint of operator satisfies the definition."""
+    x = noise_element(operator.domain)
+    y = noise_element(operator.range)
```

Comment:
Well the tests are wildly off, not like 5% or so. With that said, I changed the phantom to be a cuboid, which should be nicer.